core/jvm/src/main/scala/fs2/interop/flow/syntax.scala

```diff
@@ -0,0 +1,66 @@
+package fs2
+package interop
+package flow
+
+import cats.effect.kernel.{Async, Resource}
+
+import java.util.concurrent.Flow.{Publisher, Subscriber}
+
+object syntax {
```

Comment:
I moved the implicit classes to here from the `package` because:

1. To fix some annoying error with the doctest on `2.12`
2. IMHO, is better not to import both the explicit methods and the syntax when doing `import fs2.inteorp.flow._`
3. To follow the standard of other libraries like **cats** and **cats-effect**